shakedown/dcos/package.py

```diff
@@ -123,7 +121,7 @@ def install_package(
     if wait_for_completion:
         print("\n{}waiting for {} deployment to complete...\n".format(
             shakedown.cli.helpers.fchr('>>'), service_name))
-        if expected_running_tasks > 0:
+        if expected_running_tasks > 0 and service_name is not None:
```

Comment:
that used to be the case.. but neo4j does not have a service name and thus does not have a service end point :(